tests/Fixer/PhpUnit/PhpUnitTestAnnotationFixerTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Gert de Pagter
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitTestAnnotationFixer
+ */
+final class PhpUnitTestAnnotationFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param mixed $expected
+     * @param mixed $input
+     */
+    public function testFix($expected, $input, array $config)
+    {
+        $this->fixer->configure($config);
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * Case 1: Notation present, and wanted
+     * Case 2: Notation present, and not wanted
+     * Case 3: Notation not present, but wanted
+     * Case 4: Notation not present, but wanted, and there is already a notation
+     * Case 4: Notation not present, and not wanted
```

Comment:
hint:
create data provider like
```php
return [
    'Notation present, and wanted' => [ ... ],
    'Notation present, and not wanted' => [ ... ],
];
```

and then your descriptions will be used by PHPUnit during reporting ;)